src/directives/pagination.ts

```diff
@@ -39,8 +39,7 @@ export interface PaginationDirectiveConfiguration<T> {
 
 export const paginationDirective = <T>({table}: PaginationDirectiveConfiguration<T>): PaginationDirective => {
     let {slice: {page: currentPage, size: currentSize}} = table.getTableState();
-    let itemListLength = table.filteredCount;
```

Comment:
I took this out only because the test passed and the table instance provides `filteredCount` property. I need only the first commit to fix pageCount.